.github/workflows/node-step-azure-storage-build-and-deploy.yml

```diff
@@ -73,47 +86,62 @@ jobs:
       run:
         working-directory: ${{ inputs.WORKING_DIRECTORY }}
 
+    outputs:
+      ARTIFACT_NAME: ${{ steps.artifact-name.outputs.ARTIFACT_NAME }}
+
     env: ${{ fromJson(inputs.ENV_VARIABLES) }}
     steps:
+      - name: Generate artifact name
+        id: artifact-name
+        uses: zupit-it/pipeline-templates/.github/actions/artifact/generate-name@fix-azure-login
+        with:
+          NAME_PREFIX: node-build
+
       - name: Build
-        uses: zupit-it/pipeline-templates/.github/actions/node/build@v1.3.1
+        uses: zupit-it/pipeline-templates/.github/actions/node/build@fix-azure-login
         with:
           NODE_VERSION: ${{ inputs.NODE_VERSION }}
           RELEASE_ENVIRONMENT: ${{ inputs.RELEASE_ENVIRONMENT }}
           WORKING_DIRECTORY: ${{ inputs.WORKING_DIRECTORY }}
           BUILD_ARGS: ${{ inputs.BUILD_ARGS }}
           PROJECT: ${{ inputs.PROJECT }}
+          CHECKOUT_REF: ${{ inputs.CHECKOUT_REF }}
 
       - name: Upload build artifact
-        uses: actions/upload-artifact@v3
+        uses: zupit-it/pipeline-templates/.github/actions/artifact/upload@fix-azure-login
```

Comment:
Conviene probabilmente farlo puntare al tag corrente, in modo che alla release si incrementi :thinking: